2996_reflection/reflection.md

```diff
@@ -6280,9 +6286,9 @@ consteval info type_of(info r);
 consteval info object_of(info r);
 ```
 
-[#]{.pnum} *Constant When*: `r` is a reflection representing either an object or a variable denoting an object with static storage duration ([expr.const]).
+[#]{.pnum} *Constant When*: `r` represents either an object with static storage duration ([basic.stc.general]), or a variable associated with or referring to such an object.
 
-[#]{.pnum} *Returns*: If `r` is a reflection of a variable, then a reflection of the object denoted by the variable. Otherwise, `r`.
+[#]{.pnum} *Returns*: If `r` represents a variable, then a reflection of the object associated with or referred to by the variable. Otherwise, `r`.
```

Comment:
```suggestion
[#]{.pnum} *Returns*: If `r` represents a variable, then a reflection of the object associated with, or referred to by, the variable. Otherwise, `r`.
```

Reply:
Done